Hive.NET.Core/Configuration/Storage/IHiveStorageProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Hive.NET.Core.Configuration.Storage;
+
+public interface IHiveStorageProvider
+{
+    void UpsertHive(Guid id, Components.Hive? hive);
+    Components.Hive? GetHive(Guid id);
+    List<Components.Hive?> GetAllHives();
```

Comment:
Shouldn't the type be not nullable? If no hives are found empty list should be returned.

Reply:
u right matey